README.md

```diff
@@ -10,8 +10,11 @@ The aim is that this app can work with any tbdex liquidity node, discoverying th
 ## Running
 
 * Install Hermit https://cashapp.github.io/hermit/ (on macos you can run `brew install hermit` and then `hermit shell-hooks`)
-* Ensure you have a mobile app simulator handy (XCode on macOS and runing the Simulator app will do for example)
-* Run `flutter run` from this project to build and start the app in the simulator. Use `flutter run --dart-define=DEV_PFI=your_did_string` to run against a local tbdex liquidity node at dev time.
+* Ensure you have a mobile app simulator connected and running (XCode on macOS and running the Simulator app will do for example)
+* Ensure you have Flutter [installed](https://docs.flutter.dev/get-started/install?gclid=CjwKCAiAtt2tBhBDEiwALZuhADbPopxXHm9gGW8yIY9h4WRKKvkZxPqfqZ_iWC9_W44_r6u9LyruORoC7KcQAvD_BwE&gclsrc=aw.ds) (on macos you can run `brew install --cask flutter`)
+* Run `pub get`.
+* With your simulator running, change into your `frontend` directory, then run `flutter run` to build and start the app in the simulator. Use `flutter run --dart-define=DEV_PFI=your_did_string` to run against a local tbdex liquidity node at dev time.
```

Comment:
For this, we use hermit, which will install and manage the Flutter dependency for us. Install Flutter with brew and not using hermit could lead to some unexpected behaviors.

Reply:
Also, I would suggest doing everything inside of VSCode vs. running from the command line. I also changed the way we read in `DEV_PFI`s in my las PR so that might remove the need for the `--dart-define` step here.